settings/sample-memcache.settings.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Template for memcache configuration and SASL authentication for Acquia Cloud
+ */
+
+if ($is_ah_env) {
+  $settings['cache']['default'] = 'cache.backend.memcache';
+  $settings['memcache']['sasl'] = [
+  'username' => 'user',
+  'password' => 'password',
+];
+  // When using SASL, Memcached extension needs to be used
+  // because Memcache extension doesn't support it.
+  $settings['memcache']['extension'] = 'Memcached';
+  $settings['memcache']['options'] = [
+  \Memcached::OPT_BINARY_PROTOCOL => TRUE,
```

Comment:
@lcatlett Do you have any experience running this config prod? I would love to see or hear about any perf data from sites running this in the wild.

http://cgit.drupalcode.org/memcache/tree/README.txt?h=8.x-2.x#n268

```
Other options you could experiment with:
 + Memcached::OPT_BINARY_PROTOCOL => TRUE,
    * This enables the Memcache binary protocol (only available in Memcached
      1.4 and later). Note that some users have reported SLOWER performance
      with this feature enabled. It should only be enabled on extremely high
      traffic networks where memcache network traffic is a bottleneck.
      Additional reading about the binary protocol:
        https://raw.githubusercontent.com/memcached/old-wiki/master/MemcacheBinaryProtocol.wiki
        Note: The information on the link above will eventually be ported to
        the new wiki under https://github.com/memcached/memcached/wiki.
```

Reply:
@lcatlett still curious about this, though, despite the issues being closed if you get a moment.